src/components/object-definition.ts

```diff
@@ -27,6 +27,8 @@ export function getObjectDefinition(
     realType.flags & ts.TypeFlags.Number ||
     isArrayType(realType) ||
     realTypeName === 'HTMLElement' ||
+    realTypeName.startsWith('SVG') ||
```

Comment:
The SVG elements are complex fail with the following error:

```
Error: Multiple declarations found for symbol: addEventListener
```

Having multiple function declarations is valid TS, but unsupported by documenter. There is also no reason to deep-expand native types.

Reply:
Still recommend full name matches, just in case if we have any our own type "SVGIcon" or something